Frontend/ui-driver/src/Screens/DriverProfileScreen/Controller.purs

```diff
@@ -29,12 +29,13 @@ import PrestoDOM (Eval, continue, exit, continueWithCmd)
 import PrestoDOM.Types.Core (class Loggable)
 import Log (trackAppActionClick, trackAppEndScreen, trackAppScreenRender, trackAppBackPress, trackAppTextInput, trackAppScreenEvent)
 import Screens (ScreenName(..), getScreen)
-import Screens.DriverProfileScreen.ScreenData (MenuOptions(..))
+import Screens.DriverProfileScreen.ScreenData (MenuOptions(..), MenuOptions(LIVE_STATS_DASHBOARD)) as Data
```

Comment:
No need to import `MenuOptions(LIVE_STATS_DASHBOARD)` specifically
It is already imported in MenuOptions(..)